src/Parentheses.cpp

```diff
@@ -1,5 +1,41 @@
 #include "Parentheses.hpp"
+#include <stack>
+#include <map>
+#include <algorithm>
+#include <iostream>
 
-bool Parentheses::isBalanced(std::string s) {
-    return true;
+bool Parentheses::isBalanced(std::string stringInput) {
+    std::map<char, char> bracketsMap =
```

Comment:
to może być memberem klasy, nie musi być w funkcji, ale tak też jest ok. Tylko taka sugestia.
Sam stack podobnie.